zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -146,6 +148,7 @@ public class CompactRecordLogger {
     valueLoggers.put(ValueType.USER_TASK, this::summarizeUserTask);
     valueLoggers.put(ValueType.COMMAND_DISTRIBUTION, this::summarizeCommandDistribution);
     valueLoggers.put(ValueType.MESSAGE_CORRELATION, this::summarizeMessageCorrelation);
+    valueLoggers.put(ValueType.CLOCK, this::summarizeClock);
```

Comment:
Added custom logging for `CLOCK` commands/events to improve log readability.

For example:
```
C CLOCK PIN            - #04-> -1  -1 - pin clock to 2108-05-12T07:16:33.222 (timestamp: 4366239393222)
....
C CLOCK RESET          - #27-> -1  -1 - reset clock to system time
```

instead of the default logging:
```
C CLOCK PIN            - #21-> -1  -1 - ClockRecord {"time":4366242993222}
....
C CLOCK RESET          - #27-> -1  -1 - ClockRecord {"time":0}
```

Reply:
Simplified logging to the following:
```
C CLOCK PIN                   - #04-> -1  -1      - to 2108-05-12T07:16:33.222 (timestamp: 4366239393222)
E CLOCK PINNED            - #05->#04 K03 - to 2108-05-12T07:16:33.222 (timestamp: 4366239393222)
....
C CLOCK RESET             - #27-> -1  -1      - to system time
E CLOCK RESETTED      - #28->#27 K09 - to system time
```

As the operation we can understand from logged intent value.